site/content@beta/backend-apis/about-apis.md

```diff
@@ -0,0 +1,433 @@
+---
+weight: 20
+---
+
+# About our APIs
+
+There are two primary Chat APIs:
+
+- [Agent Chat API](https://developers.livechatinc.com/beta-docs/agent-chat-api/), which serves to join a chat as an agent.
+- [Customer Chat API](https://developers.livechatinc.com/beta-docs/customer-chat-api/), which serves to join a chat as a customer.
+
+The **Agent Chat API** contains a different set of methods than the **Customer Chat API**. We separated both APIs as they cover different use cases. 
+
+## RTM API vs. Web API
+
+Both APIs have much in common. One similarity is that they're both **Real Time Messaging APIs (RTM APIs)**, which translates to enabling real-time communication. Websocket transport allows for **pushes**, which are server-client methods used to keep application state up-to-date. In case of the Agent App and the Chat Widget, the continuous connection is crucial. That's why they both implement RTM API.
+
+|**Characteristics**|     **RTM API**      | **Web API**               |
+|:-----------------:|:--------------------:|:-------------------------:|
+|**connection type**|   websocket          |  independent requests     |
+| **makes use of**  |   pushes             |    webhooks               |
+|**implemented by** |Agent App, Chat Widget|external apps, integrations|
+
+Integrations built upon the LiveChat Platform don't usually need to keep the connection open continuously. In the context of app functioning, short delays are insignificant. It's the reason why **Web API** is more commonly used than **RTM API** when building an integration. Just like REST API, Web API is based on sending the **request** and getting the **response**. Instead of pushes, Web API makes use of **webhooks**.
+
+## Events
+
+**Events** are portions of data sent to a chat using the [send_event](https://developers.livechatinc.com/beta-docs/agent-chat-api/#send-event) method. 
```

Comment:
Brak informacji, że zostaniemy przeniesieni do innego docsa.

Reply:
Tu by się przydała ta ikonka, o której piszesz niżej. Ew. możemy napisać w nawiasie, że to jest (Agent Chat API)